docs/api/product_attributes.rst

```diff
@@ -48,6 +48,185 @@ If you request for more detailed data, you will receive an object with the follo
 
     Read more about :doc:`Product Attributes in the component docs</components/Product/models>`.
 
+Creating a Product Attribute
+----------------------------
+
+To create a new product attribute you will need to call the ``/api/v1/products-attributes/{type}`` endpoint with the ``POST`` method.
+
+Definition
+^^^^^^^^^^
+
+.. code-block:: text
+
+    POST /api/v1/product-attributes/{type}
+
++---------------+----------------+--------------------------------------------------+
+| Parameter     | Parameter type | Description                                      |
++===============+================+==================================================+
+| Authorization | header         | Token received during authentication             |
++---------------+----------------+--------------------------------------------------+
+| type          | url attribute  | Type of the product attribute (for example text) |
++---------------+----------------+--------------------------------------------------+
+| code          | request        | **(unique)** Product attribute identifier        |
++---------------+----------------+--------------------------------------------------+
+
+Example
+^^^^^^^
+
+To create a new text product attribute use the below method:
+
+.. code-block:: bash
+
+    $ curl http://demo.sylius.org/api/v1/product-attributes/text \
+        -H "Authorization: Bearer SampleToken" \
+        -H "Content-Type: application/json" \
+        -X POST \
+        --data '
+            {
+                "code": "mug_material"
+            }
+        '
+
+Exemplary Response
+^^^^^^^^^^^^^^^^^^
+
+.. code-block:: text
+
+    STATUS: 201 CREATED
+
+.. code-block:: json
+
+    {
+        "id": 1,
+        "code": "mug_material",
+        "type": "text",
+        "values": [],
+        "position": 0,
+        "translations": {},
+        "values": [],
+        "_links": {
+            "self": {
+                "href": "\/api\/v1\/product-attributes\/mug_material"
+            }
+        }
+    }
+
+.. warning::
+
+    If you try to create a product attribute without code you will receive a ``400 Bad Request`` error, that will contain validation errors.
+
+.. code-block:: bash
+
+    $ curl http://demo.sylius.org/api/v1/product-attributes/text \
+        -H "Authorization: Bearer SampleToken" \
+        -H "Content-Type: application/json" \
+        -X POST \
+        --data '
```

Comment:
I'm not sure I follow - did you mean to comment on the empty `--data` argument in the errored call below (removed in the meantime)?